decompiler/backend/variabledeclarations.py

```diff
@@ -134,4 +134,6 @@ def get_initial_value(variable: GlobalVariable) -> str:
             for requirement in operation.requirements:
                 if isinstance(requirement, GlobalVariable):
                     requirement.unsubscript()
+        if str(variable.type) == 'void *' and isinstance(variable.initial_value, int):
+            return hex(variable.initial_value)
```

Comment:
`if str(variable.type) == 'void *'` kinda feels like a hack. Any reason not to do this for all pointer types?

Reply:
None at all. Changing it